tests/phpunit/tests/Story_Renderer.php

```diff
@@ -107,4 +107,51 @@ public function test_replace_body_end_tag() {
 
 		$this->assertSame( $expected, $actual );
 	}
+
+	public function test_maybe_add_analytics() {
```

Comment:
Nit pick. It is normally better to have this as two tests. One to test if you should and when you should not insert the tag.

Reply:
@spacedmonkey Tests are updated.

Note that the gtag config also got updated based on [this doc](https://docs.google.com/document/d/1q8rkYYMKZh3R2eqkpwgGkx6QX0oSXoLYtIO6ml1BFD0/edit#), would be great if you could re-review that part.